styles.css

```
@@ -0,0 +1,662 @@
+@import url('https://fonts.googleapis.com/css2?family=Padauk&display=swap');
+
+body {
+  font-family: 'Padauk', sans-serif;
+  margin: 0px;
+  background-color: #111827;
+  color: white;
+}
+
+a:hover,
+a:visited,
+a:link,
+a:active {
+  color: white;
+}
+
```

here to display anchor tag as color `white`, you have used all pseudo-class, which seemed inefficient to me, you can do this which will give the same effect
```suggestion

a {
  color: white;
}

```

---

Done